### ISSUE TITLE
k8s-bigip-ctlr stops writing ingress policies on VIP after 20 entries

### ISSUE THREAD

**Issue by twittyc** [F5Networks/k8s-bigip-ctlr#724]
#### Description
When using name based virtual host routing on an ingress, the bigip controller stops adding node pools and ingress policy entries after 20 entries in the policy. 
#### Kubernetes Version
v1.10.5

#### Controller Version
1.4.1

#### BIG-IP Version
BIG-IP 12.1.3.1 Build 0.39.9 Engineering Hotfix

#### Diagnostic Information

Working ingress config:
```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    field.cattle.io/publicEndpoints: '[{"addresses":["10.10.10.10"],"port":80,"protocol":"HTTP","serviceName":"pim-stage:feature-ctssol-88044-pim-xlatitem","ingressName":"pim-stage:feature-ctssol-88044-pim-xlatitem","hostname":"xlatitem-feature-ctssol-88044.cernerkube.net","path":"/","allNodes":true}]'
    virtual-server.f5.com/ip: controller-default
    virtual-server.f5.com/partition: kubernetes_internal
  creationTimestamp: 2018-07-12T18:07:47Z
  generation: 1
  labels:
    app: pim-xlatitem
    chart: pim-xlatitem-1.0.0
    env: feature-ctssol-88044
    heritage: Tiller
    release: feature-ctssol-88044
  name: feature-ctssol-88044-pim-xlatitem
  namespace: pim-stage
  resourceVersion: "1757692"
  selfLink: /apis/extensions/v1beta1/namespaces/pim-stage/ingresses/feature-ctssol-88044-pim-xlatitem
  uid: 7bab4ef1-85fe-11e8-84c9-38eaa733551c
spec:
  rules:
  - host: xlatitem-feature-ctssol-88044.cernerkube.net
    http:
      paths:
      - backend:
          serviceName: feature-ctssol-88044-pim-xlatitem
          servicePort: 3000
        path: /
status:
  loadBalancer:
    ingress:
    - ip: 10.10.10.10
```

Not working Ingress config:
```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    field.cattle.io/publicEndpoints: '[{"addresses":["10.10.10.10"],"port":80,"protocol":"HTTP","serviceName":"pim-stage:feature-ctssol-88666-pim-ps-associate","ingressName":"pim-stage:feature-ctssol-88666-pim-ps-associate","hostname":"psassociates-feature-ctssol-88666.cernerkube.net","path":"/","allNodes":true}]'
    virtual-server.f5.com/ip: controller-default
    virtual-server.f5.com/partition: kubernetes_internal
  creationTimestamp: 2018-07-20T14:55:43Z
  generation: 1
  labels:
    app: pim-ps-associate
    chart: pim-ps-associate-1.0.0
    env: feature-ctssol-88666
    heritage: Tiller
    release: feature-ctssol-88666
  name: feature-ctssol-88666-pim-ps-associate
  namespace: pim-stage
  resourceVersion: "6575244"
  selfLink: /apis/extensions/v1beta1/namespaces/pim-stage/ingresses/feature-ctssol-88666-pim-ps-associate
  uid: f9c1f88c-8c2c-11e8-8879-38eaa73358a0
spec:
  rules:
  - host: psassociates-feature-ctssol-88666.cernerkube.net
    http:
      paths:
      - backend:
          serviceName: feature-ctssol-88666-pim-ps-associate
          servicePort: 3000
        path: /
status:
  loadBalancer:
    ingress:
    - ip: 10.10.10.10
```

k8s-bigip-ctrl logs:

```
2018/07/20 15:00:01 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:01 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:01 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:02 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:02 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:02 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:02 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:02 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:03 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
2018/07/20 15:00:03 [WARNING] Overwriting existing entry for backend {ServiceName:feature-ctssol-88408-pim-ps-manufacturer ServicePort:3000 Namespace:pim-stage}
```

**Comment by amudukutore** [comment]
We believe this was fixed by the performance improvements that were made as part of the v1.6.0 release - see [release-notes](https://github.com/F5Networks/k8s-bigip-ctlr/blob/master/docs/RELEASE-NOTES.rst#v160) for details. Please upgrade to the latest controller version when you have a chance to see if you still run into this issue. If its fixed, we'd like to close this out.

**Comment by dgarrisonf5** [comment]
Closing issue as this was corrected in release v1.6.0
